recipes/pyspark-flame/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyspark-flame" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 807099c77fbaefaeb71217995bd22d203c697c6ce32c8e75ff53e1b1b56d9322
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pypandoc
```

Comment:
I don't think it should be here
```suggestion
```

Reply:
I must admit I wasn't that clear after reading the documentation surrounding the requirements section. pypandoc is listed in the pyspark-flame's `setup.py` as `setup_requires=['pypandoc'],`, which I thought would have made it appropriate to add to this section?

Reply:
It should be in ``host`` then, not in ``build``
If you want to, next time you can give a go on [Grayskull](https://github.com/marcelotrevisani/grayskull) top generate recipes of packages on ``pypi``

Reply:
Yeah I just had another read through, makes a bit more sense. I'll take a look at Grayskull.

Reply:
Done.